test/trivial_auction.js

```diff
@@ -101,4 +101,28 @@ contract('TrivialToken - Auction tests', (accounts) => {
         await bidAuction();
         await finishAuction();
     })
+
+    it('check Auction buyout - one user 100% tokens', async () => {
+        assert.isAbove(parseInt(await token.getTokens.call(me)), 0);
+        assert.isAbove(parseInt(await token.getTokens.call(accounts[8])), 0);
+        assert.isAbove(parseInt(await token.getTokens.call(accounts[0])), 0);
+        assert.isAbove(parseInt(await token.getTokens.call(accounts[1])), 0);
+        assert.isAbove(parseInt(await token.getTokens.call(accounts[2])), 0);
+        assert.isAbove(parseInt(await token.getTokens.call(accounts[3])), 0);
+        await token.setTokens(me, 0);
+        await token.setTokens(accounts[8], 0);
+        await token.setTokens(accounts[0], 0);
+        await token.setTokens(accounts[1], 0);
+        await token.setTokens(accounts[2], 0);
+        await token.setTokens(accounts[3], 1000000);
+        assert.equal(parseInt(await token.getTokens.call(me)), 0);
```

Comment:
Are all these `getTokens` asserts really necessary? They take up more than 50% of this test and they in fact do not test anything meaningful.